code/modules/power/singularity/emitter.dm

```diff
@@ -71,7 +96,7 @@
 				src.active = 1
 				user << "<span class='notice'>You turn on \the [src].</span>"
 				src.shot_number = 0
-				src.fire_delay = 100
+				src.fire_delay = maximum_fire_delay

```

Comment:
nono, how it works is this:

it goes to maximum delay from first shot, which is 100 by default

it does it's 3 shot thing, and then rolls between the min and max for the delay before the next 3 shots